### ISSUE TITLE
Select all: pre element within the last child of mce-content-body get's deselected on keyup

### ISSUE THREAD

**Issue by solygen** [tinymce/tinymce#3795]
**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**

When using keyboard shortcut to trigger 'Select All' a pre element in the last child of mce-content-body get's deselected on keyup.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

- Target: https://codepen.io/anon/pen/ZyMxVP
- Fork of the basic example with a wrapping pre element added within the last block

```
<p>
  <pre> // <--
    Thanks for supporting TinyMCE! We hope it helps you and your users create great content.
    <br>All the best from the TinyMCE team.
  </pre> // <--
</p>
```

1. visit https://codepen.io/anon/pen/ZyMxVP
2. scroll down until the 'Tanks for supporting...' is scrolled into viewport
3. Press cmd+a to select the whole text
4. notice that the text get's selected first but once you release the cmd key the mentioned block get's immediately deselected
5. with enabled breakpoints in chrome dev tools the bug is not observable

**What is the expected behavior?**

4. no deselection on keyup

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

seems to be introduced with 4.5.3 and is still reproducible with latest chrome and firefox on MacOS

**Comment by solygen** [comment]
seems to be fixed with 4.6.5

**Comment by thedrjim** [comment]
This issue has been closed due to inactivity. Tiny will soon be working with members of the community to close other old issues on our GitHub repositories.

Please note the official support window for TinyMCE 4 is ending on December 31, 2020. If you haven't already upgraded to TinyMCE 5 it is highly encouraged that you do so. Upgrading will give you access to the latest releases and updates made to our WYSIWYG editor focused on bringing the latest UX and developer features into the editing experience. To assist with upgrading the editor we have put together a migration guide you can use here - [https://www.tiny.cloud/blog/how-to-migrate-from-tinymce-4-to-tinymce-5/](https://www.tiny.cloud/blog/how-to-migrate-from-tinymce-4-to-tinymce-5/)

If you have any questions or concerns, you can contact me at community-qa@tiny.cloud. You can also join the discussion on our mailing list at the following link: community-qa+subscribe@tiny.cloud.
